src/python/bcc/__init__.py

```diff
@@ -62,13 +62,47 @@ def resolve_name(self, name):
             return -1
         return addr.value
 
+class PerfType:
+    # From perf_type_id in uapi/linux/perf_event.h
+    HARDWARE = 0
+    SOFTWARE = 1
```

Comment:
@palmtenor Should this have cache and raw types as well? I asked the same thing on the previous diff, but the force push destroyed it 💣